src/base/ElementInterface.php

```diff
@@ -433,6 +433,13 @@ public function getUrl();
      */
     public function getLink();
 
+    /**
+     * Returns a string to be used in for links to edit this element.
+     *
+     * @return string|null
+     */
+    public function getLinkText();
```

Comment:
Should be renamed to `getName()`, and `craft\elements\User::getName()` should switch over to an `@inheritdoc` comment.